src/domain/event/eventCard/__tests__/EventCard.test.tsx

```diff
@@ -57,15 +57,16 @@ test('should go to event page by clicking event card', () => {
   expect(history.location.pathname).toEqual('/');
 
   userEvent.click(
-    screen.queryByRole('link', {
+    screen.getByRole('link', {
       name: translations.event.eventCard.ariaLabelLink.replace(
         '{{name}}',
-        event.name.fi
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
is this necessary? When I remove this `as any`change it still worked

Reply:
<img width="1113" alt="Screenshot 2020-12-31 at 11 29 00" src="https://user-images.githubusercontent.com/15219142/103404206-698eb780-4b5b-11eb-9d4b-05e23085c087.png">
<img width="915" alt="Screenshot 2020-12-31 at 11 28 52" src="https://user-images.githubusercontent.com/15219142/103404212-6b587b00-4b5b-11eb-9ba9-a0ee1070a4f4.png">

I think we should override that rule for test files anyway